core_contracts/governance/governance.py

```diff
@@ -53,6 +53,12 @@ def name(self) -> str:
     def getDay(self) -> int:
         return (self.now() - self._time_offset.get()) // U_SECONDS_DAY
 
+    @external(readonly=True)
+    def getVotersCount(self, name: str) -> dict:
```

Comment:
Let's also add these data to what is returned by checkVote().

Reply:
added!!